src/OpenSSL/SSL.py

```diff
@@ -28,7 +28,6 @@
     X509Name,
     X509,
     X509Store,
-    X509StoreContext,
```

Comment:
Did we really get rid of this as part of 1.0.2 deprecation? Removing a whole class is a lot... 😮

Reply:
It's no longer needed by this module, you can import it from crypto as always.

Reply:
Oh oops, thought this was where it was exported for some reason.